# HG changeset patch
# User Marko Seric <marko.seric@math.uzh.ch>
# Date 1392887517 -3600
#      Thu Feb 20 10:11:57 2014 +0100
# Branch stable
# Node ID 35100920494f23ded930ae53493fa70eb5070c27
# Parent  e4c319ed2414252c4cc6dc35639b6fc2cdf78007
doc: Update docstring for qr (bug #41641).

diff --git a/libinterp/dldfcn/qr.cc b/libinterp/dldfcn/qr.cc
--- a/libinterp/dldfcn/qr.cc
+++ b/libinterp/dldfcn/qr.cc
@@ -124,17 +124,17 @@ for overdetermined systems of equations 
 @ifnottex\n\
 @var{A}\n\
 @end ifnottex\n\
  is a tall, thin matrix).  The QR@tie{}factorization is\n\
 @tex\n\
 $QR = A$ where $Q$ is an orthogonal matrix and $R$ is upper triangular.\n\
 @end tex\n\
 @ifnottex\n\
-@code{@var{Q} * @var{Q} = @var{A}} where @var{Q} is an orthogonal matrix and\n\
+@code{@var{Q} * @var{R} = @var{A}} where @var{Q} is an orthogonal matrix and\n\
 @var{R} is upper triangular.\n\
 @end ifnottex\n\
 \n\
 If given a second argument of @qcode{'0'}, @code{qr} returns an economy-sized\n\
 QR@tie{}factorization, omitting zero rows of @var{R} and the corresponding\n\
 columns of @var{Q}.\n\
 \n\
 If the matrix @var{A} is full, the permuted QR@tie{}factorization\n\
